README.md

```diff
@@ -41,6 +50,14 @@ In order to build the installer, you'll need to install some tools.
     dist/DeveloperPlatformInstaller-win32-x64/DeveloperPlatformInstaller.exe
     ```
 
+If your npm install gets corrupted (or out of date) and you can't build, you can try cleaning leftover modules by deleting these files/folders:
+
+    ```
```

Comment:
you don't need backticks when using whitespaces... this way they are rendered, please remove them